### PR TITLE
[dxut, fx11, directxmath] ports updated

### PR DESCRIPTION
**Pull request by walbourn**
**dxut** port updated for June 2021 minor servicing release. See [GitHub](https://github.com/microsoft/DXUT/releases/tag/jun2021). Also updated to use latest DirectXMath via vcpkg.

Metadata only update for **fx11** port.

Metadata only update for **directxmath** port, mostly converting from CONTROL to vcpkg.json.

### CODE REVIEW

**Comment by walbourn** [comment]
Verified all support triplets build:

```
vcpkg install dxut:x86-windows
vcpkg install dxut:x86-windows-static
vcpkg install dxut:x64-windows
vcpkg install dxut:x64-windows-static
```

**Comment by walbourn** [comment]
Verified all supported **fx11** triplets:

```
vcpkg install effects11:x86-windows
vcpkg install effects11:x86-windows-static
vcpkg install effects11:x64-windows
vcpkg install effects11:x64-windows-static
vcpkg install effects11:arm64-windows
vcpkg install effects11:arm64-windows-static
vcpkg install effects11:x86-uwp
vcpkg install effects11:x64-uwp
vcpkg install effects11:arm-uwp
vcpkg install effects11:arm64-uwp
```

**Comment by walbourn** [comment]
Verified all *directxmath* triplets:

```
vcpkg install directxmath:x86-windows
vcpkg install directxmath:x86-windows-static
vcpkg install directxmath:x64-windows
vcpkg install directxmath:x64-windows-static
vcpkg install directxmath:arm64-windows
vcpkg install directxmath:arm64-windows-static
vcpkg install directxmath:x86-uwp
vcpkg install directxmath:x64-uwp
vcpkg install directxmath:arm-uwp
vcpkg install directxmath:arm64-uwp
vcpkg install directmath:x64-linux
vcpkg install directmath:arm64-linux
```

**Comment by walbourn** [comment]
@NancyLi1013 This PR is ready for review when you have a moment. Thanks.

File: ports/directxmath/vcpkg.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "name": "directxmath",
+  "version-string": "jan2021",
+  "port-version": 1,
+  "description": "DirectXMath SIMD C++ math library",
+  "homepage": "http://go.microsoft.com/fwlink/?LinkID=615560",
```
Comment (NancyLi1013) [inline ports/directxmath/vcpkg.json:6]:
```suggestion
  "homepage": "https://github.com/Microsoft/DirectXMath",
```
The URL will be redirected to https://github.com/Microsoft/DirectXMath, we can use it directly here.

Note: Seems you just update CONTROL file as manifest for `directxmath`, there is no update for source codes.

File: ports/dxut/vcpkg.json [hunk 2]
```diff
@@ -1,7 +1,12 @@
 {
   "name": "dxut",
-  "version-string": "11.25",
+  "version": "11.26",
   "description": "A \"GLUT\"-like framework for Direct3D 11.x Win32 desktop applications",
-  "homepage": "https://github.com/Microsoft/DXUT",
-  "supports": "windows & !uwp"
+  "homepage": "http://go.microsoft.com/fwlink/?LinkId=320437",
```
Comment (NancyLi1013) [inline ports/dxut/vcpkg.json:5]: Why do you not use the original URL https://github.com/Microsoft/DXUT for `homepage `field?

File: ports/effects11/vcpkg.json [hunk 3]
```diff
@@ -1,7 +1,10 @@
 {
   "name": "effects11",
-  "version-string": "11.26",
+  "version": "11.26",
+  "port-version": 1,
   "description": "Effects for Direct3D 11 (FX11) is a management runtime for authoring HLSL shaders, render state, and runtime variables together.",
-  "homepage": "https://github.com/microsoft/FX11",
+  "homepage": "http://go.microsoft.com/fwlink/?LinkId=271568",
```
Comment (NancyLi1013) [inline ports/effects11/vcpkg.json:6]: The same as above.

**Review by NancyLi1013 — CHANGES_REQUESTED** [review]

File: ports/directxmath/vcpkg.json (hunk 1 above)
Comment (walbourn) [reply ports/directxmath/vcpkg.json:6]: If you'd rather use direct than fwlinks, that's fine by me...

Comment (walbourn) [reply ports/directxmath/vcpkg.json:6]: DirectXMath was just a port update to vcpkg.json so I could include the extra metadata...

**Review by NancyLi1013 — APPROVED** [review]

**Comment by NancyLi1013** [comment]
LGTM, thanks for your updates @walbourn.

**Comment by walbourn** [comment]
@NancyLi1013 anything blocking this PR or is it just waiting for someone to take a look?

**Comment by NancyLi1013** [comment]
> @NancyLi1013 anything blocking this PR or is it just waiting for someone to take a look?

No, just need to merge now.

@ras0219 @ras0219-msft Could you please help merge this PR?

Thanks.

**Comment by ras0219-msft** [comment]
This all looks awesome, thanks for the continued support @walbourn :)

**Pull request merged**